### PR TITLE
KeepProyectilesTickedInUnloadedChunks Feature

### PR DESCRIPTION
**Pull request by TheCoderGamer1000**
Implemented a feature that keep the proyectiles ticked in unloaded chunks. 
Example: EnderPearls cannon works perfectly.

### CODE REVIEW

**Comment by TheCoderGamer1000** [comment]
this was adapted from <Carpet Addons> https://github.com/whoImT/carpet-addons

File: src/main/java/net/alephsmp/aleph_carpet/mixins/ServerChunkManagerMinix.java [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+import org.spongepowered.asm.mixin.Mixin;
+import org.spongepowered.asm.mixin.injection.At;
+import org.spongepowered.asm.mixin.injection.Inject;
+import org.spongepowered.asm.mixin.injection.callback.CallbackInfoReturnable;
+
+@Mixin(ServerChunkManager.class)
+public class ServerChunkManagerMinix {
+    @Inject(method = "shouldTickEntity", at = @At(value = "HEAD"), cancellable =  true)
+    private void onShouldTickEntity(Entity entity, CallbackInfoReturnable<Boolean> cir){
+        if(entity instanceof ProjectileEntity && AlephSimpleSettings.keepProjectilesTicked)
```
Comment (FracturedCode) [inline src/main/java/net/alephsmp/aleph_carpet/mixins/ServerChunkManagerMinix.java:16]: instanceof is slow; use boolean check first

File: src/main/java/net/alephsmp/aleph_carpet/mixins/ServerChunkManagerMinix.java [hunk 2]
```diff
@@ -0,0 +1,19 @@
+package net.alephsmp.aleph_carpet.mixins;
```
Comment (FracturedCode) [inline src/main/java/net/alephsmp/aleph_carpet/mixins/ServerChunkManagerMinix.java:1]: attribution to carpet-addons is required

**Review by FracturedCode — APPROVED** [review]
this will be put in a feature branch and merged to dev after minor edits have been made (by Tim probably)

**Pull request merged**